samples/stock/demo/heikinashi/demo.js

```diff
@@ -0,0 +1,44 @@
+Highcharts.getJSON('https://demo-live-data.highcharts.com/aapl-ohlcv.json', function (data) {
+    var ohlc = [],
+        dataLength = data.length,
+        i = 0;
+
+    for (i; i < dataLength; i += 1) {
+        ohlc.push([
+            data[i][0], // the date
+            data[i][1], // open
+            data[i][2], // high
+            data[i][3], // low
+            data[i][4] // close
+        ]);
+    }
```

Comment:
As far as I can see, this is not necessary. It just copies the data. We can just use `data` directly on both series.